benchmark/parse_output.py

```diff
@@ -0,0 +1,163 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of HGT tools.
+#
+
+import click
+import sys
+
+
+def parse_trex(input_f):
+    """ Parse output of T-REX version 3.6.
+
+    Parameters
+    ----------
+    input_f: string
+        file descriptor for T-REX output results
+
+    Returns
+    -------
+    number_of_hgts: string
+        number of HGTs reported by a tool, or NaN if an error occurred
+    """
+    string = 'hgt : number of HGT(s) found = '
```

Comment:
Here is where I meant the too many constants response. Currently there are only 4 parsing functions that each use a unique `string` for parsing their output files for HGTs. We will also be adding specific strings to each function for parsing gene losses, gains and donors/recipients (that's another minimum 4 strings per function). Likely, we will also be adding parsing functions for other HGT tools which will also involve unique strings for parsing HGTs. Wouldn't that be too crowded as constants at top of file?